frontend/src/modules/DashboardModule/components/RecentTable/index.jsx

```diff
@@ -15,7 +22,9 @@ function DropDownRowMenu({ row, entity }) {
   function Edit() {
     history.push(`/${entity}/update/${row._id}`);
   }
-  function Delete() {}
+  function Download() {
+    window.open(`${DOWNLOAD_BASE_URL}${entity}/${row._id}`, '_blank');
+  }
 
   return (
     <Menu style={{ width: 130 }}>
```

Comment:
can we change width to 140
or just say Download instead of Download PDF

Reply:
Alright